.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,81 @@
+name: Python package
+
+on:
+  push:
+  release:
+    types:
+      - published
+
+jobs:
+  build_wheels_linux:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'true'
+    - name: Build wheels
+      uses: pypa/cibuildwheel@v2.12.3
+      env:
+        CIBW_BEFORE_ALL: "sh linux-build.sh"
+        CIBW_BEFORE_BUILD: echo {project}/binaries
+        CIBW_SKIP: "cp36-* pp*"
+        CIBW_ARCHS_LINUX: auto64
+        CIBW_REPAIR_WHEEL_COMMAND: "LD_LIBRARY_PATH=/project/binaries auditwheel repair -w {dest_dir} {wheel}"
+    - uses: actions/upload-artifact@v3
+      with:
+        path: ./wheelhouse/*.whl
+
+  build_wheels_windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        arch:
+          - AMD64
+          - x86
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'true'
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.8' 
+    - uses: ilammy/msvc-dev-cmd@v1
+      with:
+        arch: ${{ matrix.arch }}
+    - name: Build dependencies
+      shell: cmd
+      run: |
+        python get-nasm.py ${{ matrix.arch }}
+        set path=%path%;%CD%\nasm-2.14.02
```

Comment:
Maybe this `2.14.02` string should be an argument to the `get-nasm.py` script?  Easier to maintain (?)

Reply:
No strong opinion on this. If you want I can change it.